pages/dkp/konvoy/2.1/choose-infrastructure/aws/multi-account/index.md

```diff
@@ -0,0 +1,135 @@
+
+[prerequisites](https://docs.d2iq.com/dkp/konvoy/2.1/choose-infrastructure/aws/quick-start-aws/#configure-aws-prerequisites)
+
+## Deploy DKP on AWS
+
+1. Deploy a management cluster in your AWS source account.
+2. Configure a trusted relationship between source and target accounts and create a management cluster:
+
+* **AWS:** [create Kubernetes AWS cluster](https://docs.d2iq.com/dkp/konvoy/2.1/choose-infrastructure/aws/quick-start-aws/#create-a-new-aws-kubernetes-cluster)
+
+### Step 1:
+
+DKP leverages the Cluster API provider for AWS (CAPA) to provision Kubernetes clusters in a declarative way. Customers declare the desired state of the cluster through a cluster configuration YAML file which is generated using:
+
+(AWS)
+```bash
+dkp create cluster aws --cluster-name=${CLUSTER_NAME} \
+--dry-run \
+--output=yaml \
+> ${CLUSTER_NAME}.yaml
+```
+
+### Step 2:
+Configure a trust relationship between the source and target accounts.
+
+
+***Follow all the prerequisite steps in both the source and target accounts***
+
+1. Create all policies and roles in management and workload accounts  
+   a. The prerequisite IAM policies for DKP are documented here: [white paper](https://docs.d2iq.com/dkp/konvoy/2.1/choose-infrastructure/aws/iam-policies/)
+ 
+2. Establish a trust relationship in workload account for the management account
+  
+    a. Go to your target (workload) account    
+    b. Search for the role control-plane.cluster-api-provider-aws.sigs.k8s.io    
+    c. Navigate to the Trust Relationship tab and select Edit Trust Relationship    
+    d. Add the following relationship  
+
+```bash
+{
+  "Effect": "Allow",
+  "Principal": {
+     "AWS": "arn:aws:iam::${mgmt-aws-account}:role/control-plane.cluster-api-provider-aws.sigs.k8s.io"
+  },
+  "Action": "sts:AssumeRole"
+}
+```
+
+3. Give permission to role in the source (management cluster) account to call the `sts:AssumeRole API`   
+    a. Log in to the source AWS account and attach the following inline policy to control-plane.cluster-api-provider-aws.sigs.k8s.io role
+
+```bash
+{
+  "Version": "2012-10-17",
+  "Statement": [
+     {
+       "Effect": "Allow",
+       "Action": "sts:AssumeRole",
+       "Resource": [
+"arn:aws:iam::${workload-aws-account}:role/control-plane.cluster-api-provider-aws.sigs.k8s.io"
+       ]
+     }
+  ]
+}
+```
+
+
+4. Modify the management cluster configuration file and update the AWSCluster object with following details
+
+```bash
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+kind: AWSCluster
+metadata:
+spec:
+  identityRef:
+     kind: AWSClusterRoleIdentity
+     name: cross-account-role
+…
+…
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+kind: AWSClusterRoleIdentity
+metadata:
+  name: cross-account-role
+spec:
+  allowedNamespaces: {}
+  roleARN: "arn:aws:iam::${workload-aws-account}:role/control-plane.cluster-api-provider-aws.sigs.k8s.io"
+  sourceIdentityRef:
+    kind: AWSClusterControllerIdentity
+    name: default
+```
```

Comment:
@shafiqueh - It seems to me like this needs some follow up here that explains what to do next, I think it's just this?

"After performing the above steps, any managed AWS clusters you create using the Management cluster will be provisioned in your target account."